operator/src/test/java/oracle/kubernetes/operator/steps/ManagedServersUpStepTest.java

```diff
@@ -244,6 +246,15 @@ void whenNoServerStartRequested_dontStartServers() {
     assertManagedServersUpStepNotCreated();
   }
 
+  @Test
+  void whenNoServerStartRequested_dontPrependUpdateCompleteConditionStep() {
```

Comment:
Is there a way to test the effect, here, rather than the actual steps created? That is, does the step in question create events or status updates that could be checked directly?

Reply:
This follows the pattern in other test cases in this class.